ArduCopter/mode_zigzag.cpp

```diff
@@ -444,7 +444,7 @@ bool ModeZigZag::calculate_next_dest_m(Destination ab_dest, bool use_wpnav_alt,
         next_dest_neu_m.z = wp_nav->get_wp_destination_NEU_m().z;
     } else {
         is_terrain_alt = copter.rangefinder_alt_ok() && wp_nav->rangefinder_used_and_healthy();
-        next_dest_neu_m.z = pos_control->get_pos_desired_U_m();
+        next_dest_neu_m.z = pos_control->get_pos_desired_NEU_m().z;
```

Comment:
this seems like a backwards steps.  Aren't we consolidating on using, "get_pos_desired_U_m"?  Ah, maybe you want to be using the same function as is used lower down?